webpack.config.js

```diff
@@ -83,18 +77,21 @@ module.exports = function (env = {}) {
           ],
         },
         {
-          test: /\.tsx?$/,
```

Comment:
why did you split tsx? into ts and tsx? the config looks the almost same or is there a difference because of the `loader` attribute?